tests/signing/tests.py

```diff
@@ -240,3 +241,12 @@ def test_base62(self):
         tests = [-(10**10), 10**10, 1620378259, *range(-100, 100)]
         for i in tests:
             self.assertEqual(i, signing.b62_decode(signing.b62_encode(i)))
+
+
+class TestSignerArgumentDeprecation(SimpleTestCase):
+    """Test class to test deprecation of positional arguments to signer class."""
+
+    def test_deprecation(self):
+        msg = "Providing positional arguments to Signer is deprecated."
+        with self.assertRaisesMessage(RemovedInDjango51Warning, msg):
+            signing.Signer("predictable-secret")
```

Comment:
We should add a test (with `@ignore_warnings(category=RemovedInDjango51Warning)`) to show that passing positional arguments still works, e.g.
```python
signing.Signer("secret", "/", "somesalt", "sha1", ["oldsecret"])
```